rig/machine_control/scp_connection.py

```diff
@@ -64,6 +66,10 @@ def send_scp(self, buffer_size, x, y, p, cmd, arg1=0, arg2=0, arg3=0,
         expected_args : int
             The number of arguments (0-3) that are expected in the returned
             packet.
+        timeout : float
+            Additional timeout in seconds to wait for a reply on top of the
+            default specified upon instantiation. (If None, the default is
+            used).
```

Comment:
To allow me to not change the socket timeout unless it is actually requested which I believe is good for the kernel(?)